lib/sles4sap_publiccloud.pm

```diff
@@ -110,6 +112,76 @@ sub get_promoted_hostname {
     return join("", @master);
 }
 
+=head2 sles4sap_cleanup
```

Comment:
In a net PR we can think about adding some unit testing about this in 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/t/12_sles4sap_publicccloud.t